src/azure-cli/azure/cli/command_modules/cdn/_params.py

```diff
@@ -82,6 +83,60 @@ def load_arguments(self, _):
         c.argument('content_types_to_compress', nargs='+')
         c.argument('profile_name', help=profile_name_help, id_part='name')
 
+    with self.argument_context('cdn endpoint rule') as c:
+        c.argument('rule_name', help='Name of the rule.')
+        c.argument('order', help='The order of the rule. The order number must start from 0 and consecutive.\
+                    Rule with higher order will be applied later.')
+        c.argument('match_variable', arg_group="Match Condition", help='Name of the match condition.')
```

Comment:
based on your custom.py , this argument should be `nargs='+'` and you should remove `.split(',')`

Reply:
fixed